sparrow_datums/keypoints/heatmaps.py

```diff
@@ -0,0 +1,73 @@
+from typing import TypeVar
+
+import numpy as np
+
+from sparrow_datums.keypoints.keypoints import Keypoints
+
+from ..chunk import Chunk
+from ..exceptions import ValidationError
+from ..types import FloatArray, PType
+from .keypoints import Keypoints
+
+T = TypeVar("T", bound="Heatmaps")
+
+
+class Heatmaps(Chunk):
```

Comment:
I don't think we want a dedicated `Heatmap` chunk. One of the main reasons for defining `Chunk` classes is efficient storage, but we shouldn't typically need to store the heatmaps directly. All the necessary information for a heatmap can be encoded in the `Keypoint`.

It _is_ convenient to be able to go from a heatmap to keypoints (like you're doing [here](https://github.com/sparrowml/sparrow-datums/blob/1a3e630622782966bbdcf1a2a1a89c5d9c3c3dc9/sparrow_datums/keypoints/heatmaps.py#L49)), but that can be handled as a `from_heatmap()` classmethod on the `Keypoint` class.

Reply:
My original design did not have a Heatmap class, but the problem I faced was that I couldn't convert a heatmap (in np.ndarray) form to a Keypoint class because I am already in Keypoint class (because to_keypoint would be a method in Keypoint class). I guess the solution here would be to return the keypoint as a numpy array and let the user convert it back to Keypoint class as if they initialize a band-new Keypoint. Let me know if this is confusing. If you think I am on the right path, all I have to do is time-travel on Git and use the single class design. Let me know.

Reply:
Yes that's very close to what I'm suggesting. The one difference is that you can put a `from_heatmap()` method on the `Keypoint` class. This is a `classmethod` that generates a new `Keypoint` instance from the heatmap array. We do something similar in the base chunk class with [dictionaries](https://github.com/sparrowml/sparrow-datums/blob/main/sparrow_datums/chunk.py#L201-L223). The user passes in a dict and gets a `Chunk` instance back. What we want instead is for the user to pass in a heatmap numpy array and get a Keypoint class back.

Reply:
okay, I think we are on the same page. Let's talk more when I have the old version rolled back.

Reply:
Perfect